src/HyperFormula.ts

```diff
@@ -3264,13 +3264,13 @@ export class HyperFormula implements TypedEmitter {
    *
    * @category Named Expressions
    */
-  public isItPossibleToAddNamedExpression(expressionName: string, expression: RawCellContent, scope?: string): boolean {
+  public isItPossibleToAddNamedExpression(expressionName: string, expression: RawCellContent, sheetId?: number): boolean {
```

Comment:
Can we keep the `scope` name for the argument? It does not matter if we're scoped by name or by id, the intention is IMO cleaner